app-containers/incus/autobuild/build

```diff
@@ -0,0 +1,38 @@
+abinfo "Building incus binaries..."
+mkdir "$SRCDIR"/out
+go build -v \
+    -ldflags "${GO_LDFLAGS[*]} -extldflags '-Wl,-z,relro -Wl,-z,now -specs=/usr/lib/autobuild3/specs/hardened-ld'" \
+    -o "$SRCDIR"/out/ \
+    -tags "agent" \
+    "$SRCDIR"/cmd/incus-agent/
+go build -v \
+    -ldflags "${GO_LDFLAGS[*]} -extldflags '-Wl,-z,relro -Wl,-z,now -specs=/usr/lib/autobuild3/specs/hardened-ld'" \
+    -o "$SRCDIR"/out/ \
+    -tags "netgo" \
+    "$SRCDIR"/cmd/incus-migrate/
+for tool in \
+    fuidshift \
+    incus \
+    lxc-to-incus \
+    lxd-to-incus \
+    incusd \
+    incus-benchmark \
+    incus-simplestreams \
+    incus-user; \
+do
+    go build -v \
+        -ldflags "${GO_LDFLAGS[*]} -extldflags '-Wl,-z,relro -Wl,-z,now -specs=/usr/lib/autobuild3/specs/hardened-ld'" \
+        -o "$SRCDIR"/out/ \
+        -tags "libsqlite3" \
+        "$SRCDIR"/cmd/$tool
+done
+
+install -Dvm755 -t "$PKGDIR"/usr/bin "$SRCDIR"/out/*
```

Comment:
You missed an abinfo here for "Installing Incus (is it incus or Incus?) binaries ..."

Reply:
Seems like the upstream suggests Incus